package.json

```diff
@@ -2,63 +2,72 @@
   "name": "bugzy_prealpha",
   "version": "0.0.0",
   "description": "A bugzilla client for Activity Stream",
-  "main": "main.js",
   "scripts": {
     "prettier": "prettier --write \"src/**/*.{ts,tsx,js,jsx}\" --ignore-path .gitignore",
-    "prestart": "npm run prettier && npm run tsc:prod",
-    "start": "npm run tsc:prod && concurrently -k -p --raw \"npm run parcel:dev\" \"npm run tsc:dev\" \"npm run serve\"",
+    "prestart": "npm run prettier",
+    "start": "concurrently -k npm:parcel:dev npm:serve",
     "start_prod": "node dist/server.js",
-    "parcel:dev": "parcel src/content/index.html --out-dir dist/content --public-url \"/\"",
-    "parcel:prod": "parcel build src/content/index.html --out-dir dist/content --public-url \"/\"",
-    "tsc:dev": "tsc -p tsconfig.server.json -w --outDir dist",
-    "tsc:prod": "tsc -p tsconfig.server.json --outDir dist",
+    "parcel:dev": "parcel watch",
+    "parcel:prod": "parcel build",
     "serve": "nodemon dist/server.js",
-    "prebundle": "npm run tsc:prod",
     "bundle": "npm run parcel:prod",
     "deploy": "git checkout -B deploy && npm run bundle && git add -f dist",
-    "test": "npm run lint && npm run tsc:prod && mocha -r ts-node/register src/**/*.test.js",
+    "test": "npm run lint && mocha -r ts-node/register src/**/*.test.js",
     "lint": "prettier --check \"src/**/*.{ts,tsx,js,jsx}\" --ignore-path .gitignore && eslint --ext=.js,.jsx,.ts,.tsx .",
     "prepush": "npm run lint"
   },
   "author": "k88hudson@gmail.com",
   "license": "MIT",
   "dependencies": {
-    "body-parser": "1.19.0",
+    "body-parser": "^1.20.2",
     "express": "4.17.3",
```

Comment:
I think we can upgrade express, request, and maybe some others. I initially upgraded everything and then just backpedaled from there until it mostly worked. But I think the issue I downgraded these things over was actually resolved by the `dirName` hack in server.ts.

For future reference, the issue there was that parcel v2 now does this ridiculous thing where it appends a string to `path.resolve(__dirname)` to traverse from the dist folder to the src folder. It seems like this is supposed to happen just for browser targets, and then it's supposed to be evaluated and replaced with the return value.

But in this case, it did the replacing in a node target, and then didn't evaluate it. So I was left with `path.resolve(__dirname, "../src")` in server.js which is supposed to be run in node. Which causes the server to serve the src files instead of the dist files 😂. I had to basically trick it into not noticing it, by doing `path.resolve(eval("__dirname"))` which is just...